packages/server/app/Controllers/Http/SearchUsersController.ts

```diff
@@ -147,20 +150,25 @@ export default class SearchUsersController {
             GetMyProfileInformationRequestBody.parse(rawBody);
 
         const user = await User.findOrFail(tmpAuthUserID);
-        await user.load('devices');
-        await user.load('mpeRooms');
-        await user.load('followers');
-        await user.load('following');
+        const { nickname: userNickname } = user;
+
+        //After this user model column are erased
+        await user
+            .loadCount('following')
+            .loadCount('followers')
+            .loadCount('mpeRooms')
+            .loadCount('devices');
```

Comment:
Do you think Lucid will do only one request for these four aggregates? It would be super fun 😄

Reply:
I think because if you don't regroup them in one, the last will erased previous `$extras` instance